pkg/writer/sarif_test.go

```diff
@@ -62,6 +62,7 @@ const violationTemplate = `{
                       },
                       "logicalLocations": [
                         {
+                          "decoratedName": "aws_s3_bucket.bucket",
```

Comment:
Why do we need this? We already have `name` and `kind` and those to combined are the resource id right?

Reply:
because there are complex plans where resource cannot be identified by type and name. Resources that are imported from modules or are multiple used within an array so the address could be something like `module.child_module.resource_type.resource_name[index]`

Reply:
Do you have an example sarif report where this is an issue? 

Also would this not be the [fullyQualifiedName](https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/sarif-v2.0-csprd02.html#_Toc10127915) or maybe the [address](https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/sarif-v2.0-csprd02.html#_Toc10127896) object should be used?